packages/espressocash_app/lib/features/kyc_sharing/screens/basic_information_screen.dart

```diff
@@ -0,0 +1,200 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+
+import '../../../di.dart';
+import '../../../l10n/l10n.dart';
+import '../../../ui/bottom_button.dart';
+import '../../../ui/colors.dart';
+import '../../../ui/dob_text_field.dart';
+import '../../../ui/loader.dart';
+import '../../../ui/radio_button.dart';
+import '../../../ui/snackbar.dart';
+import '../../country_picker/models/country.dart';
+import '../../country_picker/widgets/country_picker.dart';
+import '../models/document_type.dart';
+import '../services/kyc_service.dart';
+import '../widgets/document_picker.dart';
+import '../widgets/kyc_page.dart';
+import '../widgets/kyc_text_field.dart';
+
+class BasicInformationScreen extends StatefulWidget {
+  const BasicInformationScreen({super.key});
+
+  static Future<bool> push(BuildContext context) => Navigator.of(context)
```

Comment:
When I first open the page, fill in and save the data, then go back and reopen the screen, there are no data, although they are displayed in the profile page.

| | |
|---|---|
|![image](https://github.com/user-attachments/assets/c45524a4-6483-422b-b095-2b01dedc3378)|![image](https://github.com/user-attachments/assets/376ab7fc-6042-4bfb-9076-4078059abd6c)|

Reply:
updated